src/util/mount.c

```diff
@@ -79,24 +80,37 @@ int check_mounted(char *mountpoint) {
         mountpoint[mountpoint_len-1] = '\0';
     }
 
-    real_mountpoint = realpath(mountpoint, NULL); // Flawfinder: ignore
-    if ( real_mountpoint == NULL ) {
-        // mountpoint doesn't exist
-        return(retval);
+    if ( is_link(mountpoint) == 0 ) {
+        real_mountpoint = realpath(mountpoint, NULL); // Flawfinder: ignore
+        if ( real_mountpoint == NULL ) {
+            // mountpoint doesn't exists
```

Comment:
Why does that trailing 's' in exists keep popping back up?  That's a regression from pr #1317